src/schema/objects/columns.yaml

```diff
@@ -52,6 +52,17 @@ color:
     Hexadecimal. Label color for visualization.
   type: string
   unit: hexadecimal
+compcor:
+  name: '[a|t|w|c]_compcor_cor_[0-9a-zA-Z]+'
```

Comment:
Just FTR (attn @bendichter) something similarish came up in BEP032 context where for different `shape` of electrodes we would like to have columns describring the shapes, e.g.

```
name   shape   shape_radius  shape_width
1      square       n/a         1
2      circle        1         n/a
```